examples/cjs/index.cjs

```diff
@@ -25,11 +33,23 @@ const { IO, ioDevnetProcessId, Logger } = require('@ar.io/sdk');
   const observation = await arIO.getObservations({ epochIndex: 0 });
   const distributions = await arIO.getDistributions();
 
+  const arweave = Arweave.init({
+    host: 'arweave.net',
+    port: 443,
+    protocol: 'https',
+  });
+
+  const jwk = await arweave.wallets.generate();
+
+  const processId = await spawnANT({
+    signer: new ArweaveSigner(jwk),
+  });
```

Comment:
i think we can remove this from the example as well and keep them scoped to read only APIs

Reply:
@atticusofsparta please remove